package.json

```diff
@@ -19,5 +19,8 @@
   "bugs": {
     "url": "https://github.com/manicakes/react-native-icloudstore/issues"
   },
-  "homepage": "https://github.com/manicakes/react-native-icloudstore#readme"
+  "homepage": "https://github.com/manicakes/react-native-icloudstore#readme",
+  "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.17.0"
```

Comment:
If I understand things right, this should be under `peerDependencies`, not `dependencies`.
Like this:
https://github.com/hsjoberg/react-native-icloudstore/commit/a68dc9cd260b9bd36fa1f1011d60a441c0c18db7#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R24